src/Symfony/Bundle/WebProfilerBundle/Resources/config/profiler.xml

```diff
@@ -32,6 +34,9 @@
             <argument>%kernel.debug%</argument>
         </service>
 
+        <service id="web_profiler.csp_handler" class="%web_profiler.csp_handler.class%">
+        </service>
```

Comment:
I never knew how to call them and this is why I also end up calling them "monotag" :D